libs/application/templates/new-primary-school/src/fields/Review/review-groups/Child.tsx

```diff
@@ -16,18 +16,18 @@ export const Child = ({
   goToScreen,
 }: ReviewGroupProps) => {
   const { formatMessage } = useLocale()
-  const { childsNationalId } = getApplicationAnswers(application.answers)
+  const { childNationalId } = getApplicationAnswers(application.answers)
   const { children } = getApplicationExternalData(application.externalData)
 
   // Find the child name since we only have nationalId in the answers
   const selectedChild = children.find((child) => {
-    return child.nationalId === childsNationalId
+    return child.nationalId === childNationalId
   })
 
   return (
     <ReviewGroup
       isEditable={editable}
-      editAction={() => goToScreen?.('childsNationalId')}
+      editAction={() => goToScreen?.('childNationalId')}
```

Comment:
Shouldn't this be this?

```suggestion
      editAction={() => goToScreen?.('childrenMultiField')}
```

Reply:
Absolutely